wilson/test_wilson.py

```diff
@@ -159,6 +160,12 @@ def test_config_parameters(self):
         with self.assertRaises(vol.MultipleInvalid):
             # dict value must be number
             w.set_option('parameters', {'bla': 'blo'})
+        # test deprecation warning
+        with warnings.catch_warnings(record=True) as warns:
+            warnings.simplefilter("always")
+            w.set_option('parameters', {'delta': 1.})
+            self.assertEqual(len(warns), 1)
+            self.assertTrue(issubclass(warns[-1].category, DeprecationWarning))
```

Comment:
I suggest using a `FutureWarning` instead, as `DeprecationWarnings` are usually hidden from users, see https://peps.python.org/pep-0565/#additional-use-case-for-futurewarning.

Reply:
Thanks, that's a good point!